envs/safety-gymnasium/safety_gymnasium/envs/tasks/button/button_level0.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""button task 0"""
+
+
+from collections import OrderedDict
+
+import gymnasium
+import mujoco
+import numpy as np
+from safety_gymnasium.envs.assets.geoms import Buttons
+from safety_gymnasium.envs.assets.group import GROUP
+from safety_gymnasium.envs.bases import BaseTask
+
+
+class ButtonLevel0(BaseTask):
+    """A robot must press a goal button."""
+
+    def __init__(self, config):
+        super().__init__(config=config)
+
+        self.placements_extents = [-1, -1, 1, 1]
+
+        self.add_geoms(Buttons(num=4))
+
+        self.specific_agent_config()
+        self.last_dist_goal = None
+        self.buttons_timer = None
+        self.goal_button = None
+
+    def calculate_cost(self):
+        """Determine costs depending on the agent and obstacles."""
+        # pylint: disable-next=no-member
+        mujoco.mj_forward(self.model, self.data)  # Ensure positions and contacts are correct
+        cost = {}
+
+        # Sum all costs into single total cost
+        cost['cost'] = sum(v for k, v in cost.items() if k.startswith('cost_'))
+
+        return cost
+
+    def calculate_reward(self):
+        """Determine reward depending on the agent and tasks."""
+        reward = 0.0
+        dist_goal = self.dist_goal()
+        # pylint: disable-next=no-member
+        reward += (self.last_dist_goal - dist_goal) * self.buttons.reward_distance
+        self.last_dist_goal = dist_goal
+
+        if self.goal_achieved:
+            reward += self.buttons.reward_goal  # pylint: disable=no-member
```

Comment:
It seems to be related to mujoco. It is suggested whether pylintrc can be added.

Reply:
Maybe We can consider developing until a stable version is ready， then added something in pylintrc to disable these wrong warnings.